gel/_internal/_server.py

```diff
@@ -0,0 +1,1186 @@
+# SPDX-PackageName: gel-python
```

Comment:
If this is *only* needed for tests, then we should put it in a `_testbase` directory I think